### ISSUE TITLE
Face-mask detection(fasterrcnn)_230111

### ISSUE THREAD

**Issue by zzoall** [UGeunJi/Deep_Learning_Project#5]
#### 참고코드https://www.kaggle.com/code/daniel601/pytorch-fasterrcnn

1. Import
2. Image visualization
3. Dataset & Datasetload
4. Finetuning from a pretrained model (참고한 코드는 instance_segmentation를 썼는데 이유를 모르겠어서 detection.faster_rcnn 으로 다시 할 예정)
5. Train Model
6. Function to plot image (Prediction , Target)

**Comment by UGeunJi** [comment]
고생하셨어요! 
근데 조금만 더 설명해주시면 좋을 거 같아요!
내일? 12시 지났으니까 오늘?? 자세히 설명해 주세요!!

**Comment by UGeunJi** [comment]
재영님도 올려주셨는데 궁금한 게 있어요!
내일 설명 부탁드려요~~~!
